omicron/tests/test_segments.py

```diff
@@ -90,7 +90,7 @@ def test_get_frame_segments(find):
 
 
 @pytest.fixture
-def requests_mock(requests_mock):
+def requests_mock_2(requests_mock):
```

Comment:
Can this be given a more descriptive name? Something like

```suggestion
def mock_coverage_report_request(requests_mock):
```

Reply:
I think we'd also need to rename the variable on L117 as well, correct?

Reply:
I am fine with a different name but I am more concerned whether this is going to fix the problem.

It seems the parser conda-forge is using is different than what we see on my mac, at ligo cluster, and on  github.
It would seem that there is nothing necessarily with a function definition having an argument with the same name as the function. That error seems to belong to a class definition.

Reply:
I pushed the name change (in both places)